wm_dll_resources.rc

```diff
@@ -0,0 +1,88 @@
+#include "resource.h"
```

Comment:
Since the DLL is loaded into many processes, we want to keep it as small as possible, therefore I would move this to the executable or just put the hardcoded default config in the code.